README.md

```diff
@@ -21,52 +21,37 @@ Instructions are provided for bash scripts (mac)
 Ensure you have Java installed on your machine (preferably Java 8 or later). You can check this by running java -version in your command prompt or terminal.
 
 
-# 1. Installation & setup
+Installation & setup
 
-## Running a local Kafka Environment (optional)
+## 1. Running a local Kafka Environment (optional)
 
-See https://github.com/specmesh/specmesh-build/tree/main/cli#quickstart-using-docker-on-the-local-machine
+To run a local Docker based Kafka environment, run through the steps documented in the 
+[CLI quickstart](https://github.com/specmesh/specmesh-build/tree/main/cli#quickstart-using-docker-on-the-local-machine).
 
 Note - the docker environment uses the `kafka_network`
 
-Login to Github Container Registry
-```bash
-% docker login ghcr.io -u <<username>>
->> <<token>>
-``` 
```

Comment:
I've switched the CLI docker image to public, so the login is no longer required.